### PR TITLE
Adding Mastodon, removing Twitter in all versions of maintenance_changing_maintainers

### LINKED ISSUE #600: Adding Mastodon on  maintenance_changing_maintainers in English version
"por supuesto, también hay ayuda general de R si eso se ajusta a tus necesidades: Foro de la comunidad de RStudio, StackOverflow, Twitter y Mastodon #rstats, etc."

### PR DESCRIPTION
**Pull request by maelle**
Fix #600

@pedropark99 does the Portuguese change look ok? Thank you!

### CODE REVIEW

File: bookreleaseissue.pt.Rmd [hunk 1]
```diff
@@ -2,19 +2,19 @@
 repo-actions: true
 ---
 
-# Book release guidance {#bookreleaseissue}
+# Orientação para lançamento de livros {#bookreleaseissue}
```
Comment (pedropark99) [inline bookreleaseissue.pt.Rmd:5]:
```suggestion
# Orientação para o lançamento de livros {#bookreleaseissue}
```

File: templates/book-release.pt.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+### Manutenção do repositório entre lançamentos
+
+- [ ] Consulte a página de problemas para [o guia dev](https://github.com/ropensci/dev_guide/issues) e também para o repositório [de revisões de software](https://github.com/ropensci/software-review-meta/issues), procure por mudanças que ainda devem ser feitas no guia dev. Atribua os problemas encontrados no guia dev ao marco correspondente às versões, seja esta a próxima versão, ou, às versões seguintes, e.g [versão 0.3.0](https://github.com/ropensci/dev_guide/milestone/2). Encoraje novos PRs e revise eles.
+
+### 1 mês antes do lançamento
+
+- [ ] Lembre os editores de abrirem problemas/PRs para itens que desejam ver na próxima versão.
+
+- [] Execute [a função `devguide_prerelease()`](https://github.com/ropensci-org/devguider) do pacote `devguider`.
```
Comment (pedropark99) [inline templates/book-release.pt.md:11]:
```suggestion
- [ ] Execute [a função `devguide_prerelease()`](https://github.com/ropensci-org/devguider) do pacote `devguider`.
```

File: templates/book-release.pt.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+### 2 semanas antes do lançamento
+
+- [ ] Escreva um rascunho para uma postagem de blog (ou nota técnica) sobre o lançamento com antecedência suficiente para que os(as) editores(as) e, em seguida, o(a) gerente da comunidade, possam revisá-lo (2 semanas). [Exemplo](https://github.com/ropensci/roweb3/pull/291), [instruções gerais para a postagem no blog](https://blogguide.rpensci.org/), [instruções específicas para as postagens de lançamento](#releaseblogpost).
+
+- [] Crie um PR a partir da branch `dev` para a branch `master` e, em seguida, comunique aos editores através do GitHub e do Slack. Mencione o rascunho da postagem do blog em um comentário dentro deste PR.
+
+### Lançamento
+
+- [] Verifique as URLs usando [a função `devguide_urls()` do pacote {devguider}](https://github.com/ropensci-org/devguider)
```
Comment (pedropark99) [inline templates/book-release.pt.md:28]:
```suggestion
- [ ] Verifique as URLs usando [a função `devguide_urls()` do pacote {devguider}](https://github.com/ropensci-org/devguider)
```

File: templates/book-release.pt.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+- [] Verifique a ortografia usando [a função `devguide_spelling()` do pacote {devguider}](https://github.com/rpensci-org/devguider). Atualize também a [WORDLIST](https://github.com/ropensci/dev_guide/blob/master/inst/WORDLIST) conforme necessário.
```
Comment (pedropark99) [inline templates/book-release.pt.md:30]:
```suggestion
- [ ] Verifique a ortografia usando [a função `devguide_spelling()` do pacote {devguider}](https://github.com/rpensci-org/devguider). Atualize também a [WORDLIST](https://github.com/ropensci/dev_guide/blob/master/inst/WORDLIST) conforme necessário.
```

File: templates/book-release.pt.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+- [ ] Para cada contribuição/alteração verifique se as NOTÍCIAS no arquivo `Appendix.Rmd` foram atualizadas.
+
+- [ ] Planeje uma data para o lançamento e se comunique com o/a gerente da comunidade da rOpenSci, que lhe dará uma data para publicar uma postagem no blog (ou nota técnica).
+
+
+### 2 semanas antes do lançamento
+
+- [ ] Escreva um rascunho para uma postagem de blog (ou nota técnica) sobre o lançamento com antecedência suficiente para que os(as) editores(as) e, em seguida, o(a) gerente da comunidade, possam revisá-lo (2 semanas). [Exemplo](https://github.com/ropensci/roweb3/pull/291), [instruções gerais para a postagem no blog](https://blogguide.rpensci.org/), [instruções específicas para as postagens de lançamento](#releaseblogpost).
+
+- [] Crie um PR a partir da branch `dev` para a branch `master` e, em seguida, comunique aos editores através do GitHub e do Slack. Mencione o rascunho da postagem do blog em um comentário dentro deste PR.
```
Comment (pedropark99) [inline templates/book-release.pt.md:24]:
```suggestion
- [ ] Crie um PR a partir da branch `dev` para a branch `master` e, em seguida, comunique aos editores através do GitHub e do Slack. Mencione o rascunho da postagem do blog em um comentário dentro deste PR.
```

**Review by yabellini — APPROVED** [review]

**Review by mpadge — APPROVED** [review]

**Comment by maelle** [comment]
thanks @pedropark99 @mpadge @yabellini

**Pull request merged**